nservicebus/vscode.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Debugging NServiceBus in Visual Studio Code
+summary: How to configure Visual Studio Code to build and debug multiple NServiceBus endoints simultaneously
+component: Core
+reviewed: 2018-08-01
+---
+
+This article describes how to configure [Visual Studio Code](https://code.visualstudio.com/) (or "VS Code") to build an NServiceBus solution with multiple projects and debug multiple endpoints simultaneously.
+
+NServiceBus 7 and above is designed to run on .NET Core, which means instead of using Visual Studio as an IDE, users can also use the cross-platform VS Code application to build systems with NServiceBus.
+
+While Visual Studio contains specific support for .NET projects, VS Code is a more lightweight editor for any language, so some configuration is necessary for VS Code to know how to build and debug .NET projects. Because NServiceBus projects typically involve running multiple startup projects to test multiple endpoints simultaneously, a little extra configuration is needed.
+
+## Prerequisities
+
+* This article assumes knowledge of NServiceBus solutions.
+* VS Code must have the [C# for Visual Studio Code (powered by OmniSharp)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) extension installed.
+
+## Configuration files
+
+The build and debug system in VS Code is controlled by two files located in a `.vscode` directory at the project root:
+
+* _{Project Root}_
+    * `.vscode/`
+        * `launch.json`
+        * `tasks.json`
+
+The **launch.json** file describes different projects within the solution and describes how to launch each one.
+
+The **tasks.json** file describes actions to take, the most common of which is to build the code. These tasks can be executed as a prerequisite of executing a configuration in **launch.json**.
```

Comment:
Second sentence tweak:

These tasks can be configured to run before launching a configuration in **launch.json**.